analytics.go

```diff
@@ -57,7 +57,7 @@ type client struct {
 
 	// This HTTP client is used to send requests to the backend, it uses the
 	// HTTP transport provided in the configuration.
-	http http.Client
+	http *http.Client
 }
```

Comment:
So first thing was because the default http client is a pointer (https://golang.org/pkg/net/http/#pkg-variables).

Secondly, reading https://golang.org/doc/faq#Pointers "Each variable in Go exists as long as there are references to it.", so not sure why using a pointer causes more work for the GC?